i18n/en.json

```diff
@@ -1,4 +1,5 @@
 {
     "continue-reading": "Continue Reading",
-    "read-more": "Read more on Wikipedia"
+    "read-more": "Read more on Wikipedia",
+    "wikipedia-wordmark": "WikipediA"
```

Comment:
is this a common in the translatewiki community to do the wordmark translation?

Reply:
I don't know if it's common but I also don't think it is special. It should only be the localized name of the site the preview is coming from.